+++ "b/D:\\MezzanineSafe\\Mezzanine\\libincludes\\common\\bulletsrc\\bulletsvnsrc\\Extras\\Serialize\\BulletFileLoader\\bDNA.cpp"
@@ -348,7 +348,7 @@ static int name_is_array(char* name, int* dim1, int* dim2) {
 void bDNA::init(char *data, int len, bool swap)
 {
 	int *intPtr=0;short *shtPtr=0;
-	char *cp = 0;int dataLen =0;long nr=0;
+	char *cp = 0;int dataLen =0;intptr_t nr=0;
 	intPtr = (int*)data;
 
 	/*
@@ -385,9 +385,9 @@ void bDNA::init(char *data, int len, bool swap)
 		cp++;
 	}
 
-	
+
 	{
-		nr= (long)cp;
+		nr= (intptr_t)cp;
 	//long mask=3;
 		nr= ((nr+3)&~3)-nr;
 		while (nr--)
@@ -420,7 +420,7 @@ void bDNA::init(char *data, int len, bool swap)
 	}
 
 {
-		nr= (long)cp;
+		nr= (intptr_t)cp;
 	//	long mask=3;
 		nr= ((nr+3)&~3)-nr;
 		while (nr--)